### ISSUE TITLE
Visualization

### ISSUE THREAD

**Issue by jihwan1ua** [401ChemistryGenealogy/ChemistryGenealogy#13]
Visualization for the geneaology

**Comment by shikitan** [comment]
How does something like this look @slmyers 
https://github.com/marmelab/ArchitectureTree

it uses d3 nd angularjs

**Comment by slmyers** [comment]
Yeah, that looks really promising. We should definitely investigate. @shikitan
